third_party/2and3/click/testing.pyi

```diff
@@ -0,0 +1,74 @@
+from typing import (IO, Any, BinaryIO, ContextManager, Dict, Iterable, List,
+                    Mapping, Optional, Tuple, Union)
+
+from .core import BaseCommand
+
+clickpkg: Any
+
+class EchoingStdin:
+    def __init__(self, input: BinaryIO, output: BinaryIO) -> None: ...
+    def __getattr__(self, x: str) -> Any: ...
+    def read(self, n: int = ...) -> bytes: ...
+    def readline(self, n: int = ...) -> bytes: ...
+    def readlines(self) -> List[bytes]: ...
+    def __iter__(self) -> Iterable[int]: ...
+
+def make_input_stream(input: IO, charset: str) -> BinaryIO: ...
+
+class Result:
+    runner: CliRunner = ...
+    stdout_bytes: bytes = ...
+    stderr_bytes: Any = ...
+    exit_code: int = ...
+    exception: Any = ...
+    exc_info: Optional[Any] = ...
+    def __init__(
+        self,
+        runner: CliRunner,
+        stdout_bytes: bytes,
+        stderr_bytes: Any,
+        exit_code: int,
+        exception: Any,
+        exc_info: Optional[Any] = ...,
+    ) -> None: ...
+    @property
+    def output(self) -> str: ...
+    @property
+    def stdout(self) -> bytes: ...
+    @property
+    def stderr(self) -> bytes: ...
+
+class CliRunner:
+    charset: Optional[str] = ...
+    env: Mapping[str, str] = ...
+    echo_stdin: bool = ...
+    mix_stderr: bool = ...
+    def __init__(
+        self,
+        charset: Optional[str] = ...,
+        env: Optional[Mapping[str, str]] = ...,
+        echo_stdin: bool = ...,
+        mix_stderr: bool = ...,
+    ) -> None:
+        ...
+    def get_default_prog_name(self, cli: BaseCommand) -> str: ...
+    def make_env(self, overrides: Optional[Mapping[str, str]] = ...) -> Dict[str, str]: ...
+    def isolation(
+        self,
+        input: Optional[IO] = ...,
+        env: Optional[Mapping[str, str]] = ...,
+        color: bool = ...,
+    ) -> ContextManager[Tuple[BinaryIO, Union[bool, BinaryIO]]]: ...
+    def invoke(
+        self,
+        cli: BaseCommand,
+        args: Optional[Union[str, Iterable[str]]] = ...,
+        input: Optional[IO] = ...,
+        env: Optional[Mapping[str, str]] = ...,
+        catch_exceptions: bool = ...,
+        color: bool = ...,
+        mix_stderr: bool = ...,
+        **extra: Any,
+    ) -> Result:
+        ...
+    def isolated_filesystem(self) -> None: ...
```

Comment:
Should return `ContextManager[str]` (or maybe Text? haven't checked).

Reply:
Good catch!  I just fixed it to `ContextManager[str]` as you suggested (returns `str` in both Python 2 and 3).